### ISSUE TITLE
Video being misaligned in timing for 2 Videos

### ISSUE THREAD

**Issue by Himums** [olivierfriard/BORIS#783]
**Indicate the version of BORIS**
 Verify you are using the last version (http://www.boris.unito.it/pages/download.html)
8.27.1

**Describe the bug**
Hey I analyze two videos at once, looking at an animal from two angles. Sometimes the videos the differ i duration by 0.1 or 0.05 seconds. I've noticed consistently that one video is ahead by less than a second. However when I quit the app and come back, it seems to be better. However it seems I need to do this for every video I timestamp. It seems as if videos synchronicity gets a bit off. Anyone have any idea what I can to solve this and what the problem is?
Sometimes I think that the videos start at different times, due to duration differences, but after quitting the app and coming back its fixed, in synced(most times). So I'm wondering what factor is making one video sometimes ahead than the other.

**To Reproduce**
You can get two videos of the same thing from different angle. I timestamp movements. So throughout the 2-5 minute videos I'm just skipping by a few frames to slow down for the precise time a movement occurred.

**Expected behavior**
One limb of the animal (or frame in video) is .1 second ahead of the other video. So at first their limb did something at time 4:30:550 but then another video has that limb do the same thing at 4:30:650(min:seconds:fraction of second)

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [windows 10, Ubuntu 22.04 ... ]
  Windows 10
 - Type of BORIS instalaltion (setup or portable ...)
 Setup

**In case of crash** attach or copy the content of the boris.log file (before relaunching BORIS).
(see http://www.boris.unito.it/pages/report_a_bug.html for its location)

**Comment by olivierfriard** [comment]
hi,
did you use a time offset during the observation creation?

**Comment by Himums** [comment]
No, I've only done that 5x(Ive gone 100s of videos). Sometimes it actually offsets, but most of the times it didn't offset anything.

**Comment by olivierfriard** [comment]
If your videos begin at a different time you have to use the offset parameter for their synchronization.
See https://www.boris.unito.it/user_guide/observations/#observation-from-media-files for details

**Comment by Himums** [comment]
Sometimes the offset doesn't do anything though, am I supposed add in anything when using it?

**Comment by christine-barry** [comment]
Hey Oli, I am having a similar problem, but with time drift! So a video that is 29 minutes long is showing up as 33 minutes by the end of the observation. I am then exporting 1980 rows to binary data instead of the expected 1800, creating problems when I match behavioural data with movement data from different sensors. Could it be the same problem? I have a short 1 minute video and the timing seemed okay for that? Got any suggestions? 

Thanks
Chris

**Comment by christine-barry** [comment]
Hey Oli, I've realised my video converter caused the time drift and gave me 35 minute MP4 videos from the 30 minute h265 videos. Sorry! C

**Comment by alisabak** [comment]
Hi! I have a similar problem with 2 videos opened in 2 players. When I score them in a frame-by-frame mode, at some point videos become unsynchronized. If I jump to another time point in the video and back, they get synchronized again. The problem is unpredictable, it appears and disappears randomly during scoring frame-by-frame. I do not use the offset; recoding the videos with a lower bitrate did not help.

I am using Windows 10. Videos are quite large (1748x1058, 20 min at 30 fps).

**Comment by christine-barry** [comment]
I converted my files to MP4 with 'handbrake' which caused the frame-rate issue. Once I converted them again through VLC my issue was fixed and I didn't have additional problems with BORIS. 
C
